pkgs/default.nix

```diff
@@ -1 +1,3 @@
-final: prev: { }
+{ nixos }: final: prev: {
+  myLib = import ../lib { inherit nixos; pkgs = final; };
```

Comment:
This falls just short of proper name spacing with e.g. the username or the environment name.

This resulted while splitting the impetus behind #145 in its individual parts.

We can still adopt proper name spacing later, since this makes the overlay more useful when shared.